### DISCUSSION TITLE
(我的) Golang 错误处理最佳实践

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#344] [Announcements]
# (我的) Golang 错误处理最佳实践

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2020/05-go-error-handling/

**Comment by x1nchen** [comment]
hi, 提案和规范 （Proposal: Error Handling 与 Spec: Error Handling） 的链接失效了。BTW，这是一篇很有价值的总结，谢谢~

**Reply by Xuanwo** [reply]
啊，我修一下，感谢反馈。正确的链接应该是 https://github.com/beyondstorage/go-storage/blob/master/docs/rfcs/11-error-handling.md

**Reply by x1nchen** [reply]
thanks~
